Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -625,7 +650,7 @@ function ViewPanelHousing( data:table )
     --Improvements
     CQUI_BuildHousingBubbleInstance("ICON_IMPROVEMENT_PASTURE", CQUI_HousingFromImprovements, "LOC_IMPROVEMENT_NAME");    -- CQUI real housing from improvements value
     --Era
-    CQUI_BuildHousingBubbleInstance("ICON_GREAT_PERSON_CLASS_SCIENTIST", data.HousingFromStartingEra, "LOC_ERA_NAME");
+    CQUI_BuildHousingBubbleInstance("ICON_GREAT_PERSON_CLASS_SCIENTIST", data.HousingFromStartingEra, "LOC_GAME_START_ERA");
```

Comment:
unmodified only shows this if value is greater than 0 (which I would assume it to be more than zero if picking anything other than ancient era?)

Reply:
Not sure what that comment means?